src/modules/wallet/components/connectButton/index.tsx

```diff
@@ -14,3 +14,58 @@ export const ConnectButton: React.FC<ConnectButtonProps> = ({ handleClick }) =>
     </button>
   );
 };
+
+interface LaceButtonProps {
+  handleClick: () => void;
+}
+export const LaceButton: React.FC<LaceButtonProps> = ({ handleClick }) => {
+  const cardano = window?.cardano;
```

Comment:
I don't think it would works on mobile versions but I think it would be fine for now